lemur/notifications/messaging.py

```diff
@@ -152,33 +152,30 @@ def send_expiration_notifications(exclude):
                     certificate
                 ).data
                 notification_data.append(cert_data)
-                security_data.append(cert_data)
 
-            if send_default_notification(
-                    "expiration", notification_data, [owner], notification.options
-            ):
-                success += 1
+            email_recipients = security_email + [owner]
+            if notification.plugin.slug == "email-notification":
+                email_recipients = notification.plugin.get_recipients(notification.options, email_recipients)
+                if send_plugin_notification(
+                        "expiration", notification_data, email_recipients, notification
+                ):
+                    success += len(email_recipients)
+                else:
+                    failure += len(email_recipients)
             else:
-                failure += 1
-
-            recipients = notification.plugin.filter_recipients(notification.options, security_email + [owner])
-
-            if send_plugin_notification(
-                "expiration",
-                notification_data,
-                recipients,
-                notification,
-            ):
-                success += 1
-            else:
-                failure += 1
-
-            if send_default_notification(
-                "expiration", security_data, security_email, notification.options
-            ):
-                success += 1
-            else:
-                failure += 1
+                if send_default_notification(
+                        "expiration", notification_data, email_recipients, notification.options
+                ):
+                    success += len(email_recipients)
+                else:
+                    failure += len(email_recipients)
+
+                if send_plugin_notification(
```

Comment:
would it make any difference if for all plugins this is called?
```
if send_plugin_notification(
                        "expiration", notification_data, email_recipients, notification
                ):
``` 


sns and slack plugins would be ignoring `email_recipients` anyways, right?

Reply:
That's true, it makes no difference. The API is a bit odd since only the email plugin needs recipients, and the others accept but ignore them. We might be able to condense down to two send calls (one plugin, one default) if I do it that way, let me double check and make sure it works.